src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -640,7 +595,7 @@ public IEnumerable<Project> GetProjectsWithInstalledPackage(Solution solution, s
         public bool CanShowManagePackagesDialog()
             => TryGetOrLoadNuGetPackageManager(out _);
 
-        private bool TryGetOrLoadNuGetPackageManager(out IVsPackage nugetPackageManager)
+        private bool TryGetOrLoadNuGetPackageManager([NotNullWhen(true)] out IVsPackage? nugetPackageManager)
```

Comment:
Is this function still necessary with the new service?

Reply:
yes.  The new service currently only exposes a single async entrypoint for getting installed packages.  The code you're looking at is about loading hte Package Manager window, which still has no other way to activate it.